bin/makisu/cmd/pull.go

```diff
@@ -41,9 +43,15 @@ func getPullCmd() *pullCmd {
 		return nil
 	}
 	pullCmd.Run = func(cmd *cobra.Command, args []string) {
+		if err := pullCmd.processFlags(); err != nil {
+			log.Errorf("failed to process flags: %s", err)
+			os.Exit(1)
+		}
+
 		pullCmd.Pull(args[0])
 	}
 
+	pullCmd.PersistentFlags().StringVar(&pullCmd.registryConfig, "registry-config", "", "Set build-time variables")
```

Comment:
Can you adjust the help text to be something like:

"Supply path to registry configuration file"

Reply:
Agreed, that makes a lot more sense to me too.

I used that wording to be consistent with the `makisu build` and `makisu push` commands that also define that option, so I also updated the help text there to what you suggested.